AlphaWallet/Core/Enjin/API.swift

```diff
@@ -0,0 +1,798 @@
+// @generated
```

Comment:
Can you write notes for how to generate this file and can we generate it with a different file name? (mention Enjin)

Reply:
Models generations comes from script `Apollo` running before building proj:
it conrains several commented lines:

```
#NOTE: make sure next lines commented, the GetEnjinToken request has bug in scheme it declares available to mint value as string but returns double instead. Make sure property changed with right type. Or ignore it at all.
#Generate models out fo graphql requests
#"${SCRIPT_PATH}"/run-bundled-codegen.sh codegen:generate --target=swift --includes=./**/*.graphql --localSchemaFile="schema.json" API.swift
#Download graphql scheme 
#"${SCRIPT_PATH}"/run-bundled-codegen.sh schema:download --endpoint="https://cloud.enjin.io/graphql/default" API.swift
```
there some inconsistencies between what api returns and what is returns in scheme, so we got types missmatch during parsing models. that is why we commened these lines